### PR TITLE
Promote CI tool neutrality.

### LINKED ISSUE #78: Update composer.json description, tags... to better represent current status
This has called my attention viewing the package @ packagist:

> Assists with running Moodle plugins in Travis CI

We should update the description to express the "ci-neutrality" of the tool. For example the description @ github is really better, IMO (though sure it also can be improved):

> Helps running Moodle plugins analysis checks and tests under various CI environments.

So that's all. Description, tags... anything that can better represent the package via `package.json`

### PR DESCRIPTION
**Pull request by kabalin**
Fixes #78

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 [![License](https://poser.pugx.org/moodlehq/moodle-plugin-ci/license)](//packagist.org/packages/moodlehq/moodle-plugin-ci)
 
 The goal of this project is to facilitate the running of tests and code
-analysis against a Moodle plugin in CI tools, such as [Travis
+analysis against a Moodle plugin using CI environment, such as [Travis
```
Comment (stronk7) [inline README.md:8]: "a CI environment" or "CI environments", but current is strange, isn't it?

**Review by stronk7 — CHANGES_REQUESTED** [review]
In general, we should, some day, also be able to remove (rename) the `moodle-local_travis` fixture and all cases there, but that's not critical at all.

Maybe we can leave the related issue open to kill those remaining cases?

**Comment by kabalin** [comment]
Thanks @stronk7  for review. I added renaming commit, so we won't delay it.

File: README.md (hunk 1 above)
Comment (kabalin) [reply README.md]: Fixed

**Review by stronk7 — DISMISSED** [review]

**Comment by stronk7** [comment]
Changes look ok. Once you get Travis tests passing... feel free to self-merge this towards the needed release (I'll be AFK the very next ours).

Thanks!

**Review by stronk7 — APPROVED** [review]

**Pull request merged**